vignettes/VectorControl.Rmd

```diff
@@ -15,117 +15,124 @@ knitr::opts_chunk$set(
-
-peak <- peak_season_offset(sprayingparams)
-spraying_events = data.frame(
-  timestep = c(1, 2) * year + peak - 3 * month,
-  name=c("Spraying 1", "Spraying 2")
-)
-
-sprayingparams <- set_spraying(
-  sprayingparams,
-  timesteps = spraying_events$timestep,
-  coverages = rep(.8, 2),
-  ls_theta = matrix(c(2.025, 2.025), nrow=2, ncol=1),
-  ls_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1),
-  ks_theta = matrix(c(-2.222, -2.222), nrow=2, ncol=1),
-  ks_gamma = matrix(c(0.008, 0.008), nrow=2, ncol=1),
-  ms_theta = matrix(c(-1.232, -1.232), nrow=2, ncol=1),
-  ms_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1)
-)
+### Plot prevalence
+```{r, fig.align='center', fig.height=5, fig.width=7}
+plot_prev()
+```
+  
+## Visualisation 
+
+It is important to understand the difference between the input `coverages` argument of `malariasimulation::set_bednets()` and the resulting population bednet usage over time in the model. In the above example, bednets are distributed to a specified percentage of the population at each distribution round; however, the level of bednet usage may be different. 
+  
+The average population bednet usage will be influenced by:   
+
+* The size and frequency of distributions specified in `set_bednets()`  
+* The assumed net retention half life (`gamman`)
+* Correlations in the recipients of nets between rounds  
+  
+The output from `malariasimulation::run_simulation()` has a variable `n_use_net` that shows the number of people using bednets at any given timestep. We can visualise the proportion of the population using bednets over time to understand how bednet usage changes. 
+
+```{r, fig.align='center', fig.height=5, fig.width=7}
+output$prop_use_net <- output$n_use_net / human_population
+
+plot(x = output$timestep, y = output$prop_use_net, type = 'l', 
+     col = cols[3], lwd = 2.5, ylim = c(0,1),
+     xlab = 'Timestep (days)', ylab = 'Proportion of population using bednets',
+     xaxs = "i", yaxs = "i", bty = "l")
+grid(lty = 2, col = 'grey80', lwd = 1)
+axis(side = 1, lty = 1, col = "black", pos = 0); axis(side = 2, lty = 1, col = "black")
+```
+  
+## Using the `netz` package to estimate coverage inputs needed to achieve target population usage
 
-output <- run_simulation(sim_length, sprayingparams)
+The [`netz` package](https://mrc-ide.github.io/netz/index.html) is a useful tool to help set up bednets in `malariasimulation`. `malariasimulation` takes as input the proportion of the population who were distributed a bednet at specific time points, but available data for a specific region or country do not always have this information. `netz` allows for simple conversions between bednet metrics like crop, access, distribution, and usage.  
+  
+See the vignettes from the `netz` package that explain these concepts more comprehensively:  
+  
+* [This vignette](https://mrc-ide.github.io/netz/articles/overview.html) explains how to estimate the distribution coverage from a specified bednet usage.   
```

Comment:
I don't think we want to include this first point and link in this vignette as a lot of that is unrelated to the malariasimulation implementation. 

The second point is enough 👍